app/components/header_component.html.slim

```diff
@@ -6,11 +6,7 @@
         = image_tag 'dfe-logo-alt.png', class: 'dfe-logo-hover', alt: 'DfE homepage'
     #service-name
       = link_to service_name, service_url, class: 'dfe-header__link dfe-header__link--service'
-    #content-header.dfe-header__content.dfe-header-f-dtonly.dfe-header-f-toplinks
-      .dfe-header__menu
-        button#toggle-menu.dfe-header__menu-toggle aria-controls='header-navigation' aria-expanded='false' data-action='click->reveal#toggle'
-          | Menu
-    / This logic has been commented out as it is not required currently
+    / Original header template
```

Comment:
`/!` this is the syntax I was concerned about to avoid making it visible the browser. I would recommend leveraging the single `/` where possible when it is all indented.

Reply:
I have updated this in the next commit to use a single `/` for the comment